src/screens/NanoContract/NanoContractTransactionsScreen.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+} from 'react-native';
+import { t } from 'ttag';
+
+import HathorHeader from '../../components/HathorHeader';
+import { NanoContractTransactionsList } from '../../components/NanoContract/NanoContractTransactionsList.component';
+import OfflineBar from '../../components/OfflineBar';
+import { COLORS } from '../../styles/themes';
+
+/**
+ * Presents a list of Nano Contract transactions.
+ */
+export function NanoContractTransactionsScreen({ navigation, route }) {
```

Comment:
I think the best name here would be `NanoContractDetails` because we don't only have the transactions; we have all the details and the history. What do you think?

Reply:
Agreed! I'll change it.

Reply:
Changed.